x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -138,6 +138,14 @@ export interface VulnScoreTrend {
   high: number;
   medium: number;
   low: number;
+  score_by_cloud_account?: Array<{
+    cloudAccountId: string;
+    cloudAccountName: string;
+    critical: number;
+    high: number;
+    medium: number;
+    low: number;
+  }>;
```

Comment:
a word regarding the design
the name `score_by_cloud_account` suggests it is a map and not an array.
A benefit of having it a map between `cloudAccountId` to its stats (I don't think the term `score` is what we calculate here) would be that you spare a search over an array whenever you pick an account.

I'd suggest fixing it. Align the structure to its name - change it to a map, and I suggest to replace the name to be more descriptive, like: `vulnerabilities_stats_by_cloud_account`